### PR TITLE
Tron roadmap breakdown

### PR DESCRIPTION
**Pull request by TronBlack**

### CODE REVIEW

File: roadmap/rewards/README.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+These rpc calls are added in support of rewards:
+
+reward QTY [RVN|TOKEN] TARGET_TOKEN [exception address list]
+
+#### Examples
+
+Example 1 (simple):
+```
+4 holders of TRONCO (unit 0) (100 Issued)
+RBQ5A9wYKcebZtTSrJ5E4bKgPRbNmr8M2H   10 TRONCO
```
Comment (cfox) [inline roadmap/rewards/README.md]:
```suggestion
RBQ5A9wYKcebZtTSrJ5E4bKgPRbNmr8M2H   10 FOXYCORP
```

File: roadmap/rewards/README.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+#### GUI - Mobile
+
+Mobile will not initially have the rewards feature.
+
+#### RPC
+
+These rpc calls are added in support of rewards:
+
+reward QTY [RVN|TOKEN] TARGET_TOKEN [exception address list]
```
Comment (cfox) [inline roadmap/rewards/README.md]: Might want to move this up above the Reward Calculation section -- it's a little hard to follow without seeing this first.

File: roadmap/messaging/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Create a broadcast message with optional expiration date.
+* Phase 1 - Enter IPFS hash of message
+* Phase 2 - Enter message, submit via POST to publish IPFS hash, submit message transaction by sending owner or channel token to same address.
+
+#### RPC
+
+These rpc calls are added in support of messaging:
+
+issue_channel TOKEN CHANNEL_NAME
```
Comment (cfox) [inline roadmap/messaging/README.md]: There is no line break here.
```suggestion
issue_channel TOKEN CHANNEL_NAME . 
```

Comment (cfox) [inline roadmap/messaging/README.md]: Need a line break here.
```suggestion
issue_channel TOKEN CHANNEL_NAME  
```

File: roadmap/messaging-antispam/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Messaging - Anti-spam
+
+Ravencoin will display messages for all tokens that you own.  But wait, what if I get an unsolicited HERPES token because they found my crypto address on-chain?  Maybe I don’t want messages from the HERPES token issuer.
+
+The holder of the owner token, e.g. ATOKEN! can send messages to holders their token ATOKEN.   The owner token has an exclamation point.  The owner token holder can also create channel tokens like ATOKEN~News, or ATOKEN~VoteInfo, or ATOKEN~Emergency   These are message channel tokens and the holder of these tokens can broadcast a message by attaching a special file (IPFS) to the transaction while sending the message channel token to themselves.
```
Comment (cfox) [inline roadmap/messaging-antispam/README.md]: These tildes are resulting in some weird text formatting..

File: roadmap/rewards/README.md (hunk 2 above)
Comment (TronBlack) [reply roadmap/rewards/README.md]: I added an example usage - near the top.

File: roadmap/messaging-antispam/README.md (hunk 4 above)
Comment (TronBlack) [reply roadmap/messaging-antispam/README.md]: Fixed.  Tilde needed to be escaped.
